msupply-foundation/open-msupply
# 2944 inbound return from outbound shipment

Fixes #2944

# 👩🏻‍💻 What does this PR do? 
 

Adds the first step of the Inbound/Customer return modal from an OutboundShipment.

![Screenshot 2024-02-15 at 12 04 22 PM](https://github.com/msupply-foundation/open-msupply/assets/55115239/790e6cc5-1f2e-424a-b53a-f5b87c36469d)


# 🧪 How has/should this change been tested? 


- Navigate to details page of existing Outbound Shipment
- Select some lines
- Click the process return action
- See the modal - you can edit the returned quantity

## 💌 Any notes for the reviewer?


Went with `InboundReturn` rather than `CustomerReturn` naming here... we should definitely talk to Richard before deciding either way, just thought this would be a nice chance to see how it looks/feels?

Also what would you call the action here? This seems long...
![Screenshot 2024-02-15 at 12 02 13 PM](https://github.com/msupply-foundation/open-msupply/assets/55115239/211b09c1-2664-4fed-a3e3-1f0e2f5c2651)
